### PR TITLE
Updated configuration, and sample simulation class

### PR DESCRIPTION
**Pull request by Johanpdrsn**
This pull request contains a working setup of configs, and a slight change to folder structure.

It also contains a very simple Simulation class, mostly just to get some feedback if this is the right direction.

### CODE REVIEW

File: src/Simulation.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    startTime: Date;
+    stopTime: Date;
+
+    constructor(input: string) {
+        this.graph = parser.parse(input);
+        this.users = [];
+        this.id = 1;
+    }
+
+    executeEvent(event: any) {
```
Comment (rafael-21701451) [inline src/Simulation.ts]: There is already an Event class defined, so we can use that instead of any

File: src/Simulation.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+class Simulation {
+    graph: DCRGraph;
+    isRunning: boolean = false;
+    id: number;
+    log: any;
+    users: any[];
+    startTime: Date;
+    stopTime: Date;
+
+    constructor(input: string) {
```
Comment (rafael-21701451) [inline src/Simulation.ts:13]: I see you don't instantiate a Log with the simulation, is it because we still haven't defined it?

**Review by rafael-21701451 — COMMENTED** [review]
Looks good to me, left some small comments that you might want to look at :)

**Comment by Johanpdrsn** [comment]
Sorry if i wasn't clear. The Simulation class was just a quick mock up to show intent, what the real commit had was the new configuration of the config files + the setup with a src and js folder :)

**Comment by rafael-21701451** [comment]
@Johanpdrsn ohhhh ok got it, since you titled it simulation I thought it was the class only! You have my approval then, go ahead and merge 👍

**Review by rafael-21701451 — APPROVED** [review]

**Comment by Johanpdrsn** [comment]
@rafael-21701451 yeah sorry, my branch was named simulation so i just auto named it, but i will change it

**Pull request merged**